frontend/amundsen_application/static/js/components/ResourceListItem/TableListItem/index.tsx

```diff
@@ -24,9 +24,26 @@ export interface TableListItemProps {
   disabled?: boolean;
 }
 
-export const getLink = (table, logging) =>
-  `/table_detail/${table.cluster}/${table.database}/${table.schema}/${table.name}` +
-  `?index=${logging.index}&source=${logging.source}`;
+export const getName = (table) => {
+  /*
+  this function get's the table name from the key to preserve original
+  capitalization since search needs the names to be lowercase for analysis
+  */
+  const splitKey = table.key.split('/');
+  const keyName = splitKey[splitKey.length - 1];
+  if (keyName.toLowerCase() === table.name) {
+    return keyName;
+  }
+  return table.name;
+};
+
+export const getLink = (table, logging) => {
```

Comment:
I would move this into a utils file, navigationUtils.ts will probably the best option.
I would also add a test for it.

Reply:
this is consistent with all of the other resource list item files

Reply:
OK, could you add a test though?